appservice/src/createAppService/createAppService.ts

```diff
@@ -24,13 +27,67 @@ export async function createAppService(
     credentials: ServiceClientCredentials,
     subscriptionId: string,
     subscriptionDisplayName: string,
-    showCreatingNode?: (label: string) => void): Promise<Site> {
+    showCreatingNode?: (label: string) => void,
+    fsPath?: string,
+    streamlinedSteps: boolean = false /* enables streamlined app creation that emulates az webapp up's behavior */): Promise<Site> {
```

Comment:
We want 'streamlined' to be the default user experience now, right? If so I would flip all of the naming to match that. For example in this case you would having something like 'showAllSteps' or 'ignoreDefaults' or 'beReallyAnnoying' idk. Honestly this should probably match whatever user setting we decide on that lets users 'not' streamline stuff